src/pages/core-candy-machine/guides/create-a-core-candy-machine-ui.md

```diff
@@ -0,0 +1,789 @@
+                "assetBurn": {
+                    "__option": "None"
+                },
+                "assetMintLimit": {
+                    "__option": "None"
+                },
+                "assetBurnMulti": {
+                    "__option": "None"
+                },
+                "assetPaymentMulti": {
+                    "__option": "None"
+                },
+                "assetGate": {
+                    "__option": "None"
+                },
+                "vanityMint": {
+                    "__option": "None"
+                }
+            }
+        },
+    ]
+}
+```
+{% /totem-accordion  %}
+
+{% /dialect %}
+{% /dialect-switcher %}
+
+### Fetch additional Candy Machine related Accounts
+The choice of Guards you implement may necessitate fetching additional accounts. For instance, if you plan to verify a wallet's minting eligibility and are utilizing the `mintLimit` Guard, you would need to retrieve the `mintCounter` account. This account maintains a record of how many NFTs a particular wallet has already minted under that specific guard.
+
+#### `MintLimit` Accounts
+When the [`MintLimit`](/core-candy-machine/guards/mint-limit) guard is active, it's advisable to retrieve the `MintCounter` for the user's wallet. This allows you to check whether the user has reached their minting limit or if they're still eligible to mint more.
+
+The following code snippet demonstrates how to fetch the `MintCounter`. Note that this example assumes you've already obtained the Candy Machine and Candy Guard data:
+
+```ts
+import { safeFetchMintCounterFromSeeds } from "@metaplex-foundation/mpl-core-candy-machine";
+
+const mintCounter = await safeFetchMintCounterFromSeeds(umi, {
+  id: 1, // The mintLimit id you set in your guard config
+  user: umi.identity.publicKey,
+  candyMachine: candyMachine.publicKey,
+  candyGuard: candyMachine.mintAuthority,
+});
+```
+
+#### `NftMintLimit` Accounts
+Similar to the `MintLimit` guard it can make sense to fetch the `NftMintCounter` of the [`NftMintLimit`](/core-candy-machine/guards/nft-mint-limit) guard to be able to verify eligibility.
+
+The following code snippet demonstrates how to fetch the `NftMintCounter`. Note that this example assumes you've already obtained the Candy Machine and Candy Guard data:
+
+```ts
+import { 
+  findNftMintCounterPda,
+  fetchNftMintCounter
+ } from "@metaplex-foundation/mpl-core-candy-machine";
+
+const pda = findNftMintCounterPda(umi, {
+  id: 1, // The nftMintLimit id you set in your guard config
+  mint: asset.publicKey, // The address of the nft your user owns
+  candyGuard: candyMachine.mintAuthority,
+  candyMachine: candyMachine.publicKey,
+});
+      
+const nftMintCounter = fetchNftMintCounter(umi, pda)
+```
+
+#### `AssetMintLimit` Accounts
+Similar to the `NftMintCounter` guard it can make sense to fetch the `AssetMintCounter` of the [`AssetMintLimit`](/core-candy-machine/guards/asset-mint-limit) guard to be able to verify eligibility.
```

Comment:
```suggestion
Similar to the `NftMintCounter` guard it can make sense to fetch the `AssetMintCounter` account of the [`AssetMintLimit`](/core-candy-machine/guards/asset-mint-limit) guard to verify eligibility.
```